samples/04_gis_analysts_data_scientists/translating_story_map_from_one_language_to_another.ipynb

```diff
@@ -0,0 +1,4030 @@
+{
```

Comment:
<p>BeautifulSoup library is an extra dependency that doesn't come in with dl installer? If so , we should provide a command in perquisite section for installing it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Added this line in the prerequisite section. Let me know if this make sense or any changes are needed for this. </p><p> </p><p>[Beautiful Soup](https://anaconda.org/anaconda/beautifulsoup4) python library to pull text out from the HTML content of the story map.</p>

Reply:
<p>Made the necessary changes</p>